content/docs/12-enterprise-version/06-air-gap-repo.md

```diff
@@ -1,108 +1,730 @@
+
+The following diagram is a high-level overview of the order of operations required to deploy a self-hosted instance of Palette in an airgap environment.
+
+
+![An architecture diagram outlining the five different event milestons](/enterprise-version_air-gap-repo_overview-order-diagram.png)
+
+
+The airgap installation can be simplified into five major milestones. 
+
+<br />
+
+1. The first milestone is to download the Open Virtual Appliance (OVA) image and deploy the instance hosting the private repository supporting the airgap environment.
+
+
+2. The private Spectro Cloud repository is initialized, and all the Palette-required artifacts are downloaded and available.
+
+
+3. The Palette Install OVA is deployed, configured, and initialized.
+
+
+4. The scale-up process to a highly available three-node installation begins.
+
+
+5. Palette is ready for usage.
+
+
+This guide will focus on the first two milestones as the remaining ones are covered in the [Migrate Cluster to Enterprise](/deploying-an-enterprise-cluster) guide and the [Install Using Quick-Start Mode](/enterprise-version/deploying-the-platform-installer) guide.
+
+
+# Prerequisites
+
+* The following minimum resources are required for deploying Palette.
+    * 2 vCPU
+    * 4 GB of Memory
+    * 100 GB of Storage. Storage sizing depends on your intended update frequency and data retention model. <br /> <br />
+
+* Ensure the following ports allow inbound network traffic. 
     * 80
     * 443
     * 5000
     * 8000
 
 
-<InfoBox>
-Storage sizing depends on your intended update frequency and data retention model.
-</InfoBox>
+* Request the Palette self-hosted installer image and the Palette air gap installer image. To request the installer images, please contact our support team by sending an email to support@spectrocloud.com. Kindly provide the following information in your email:
+
+    - Your full name
+    - Organization name (if applicable)
+    - Email address
+    - Phone number (optional)
+    - A brief description of your intended use for the Palette Self-host installer image.
+
+Our dedicated support team will promptly get in touch with you to provide the necessary assistance and share the installer image. 
+
+If you have any questions or concerns, please feel free to contact support@spectrocloud.com.
+
+<br />
+
+
+# Deploy Air Gapped Appliance
+
+
+1. Log in to vCenter Server by using the vSphere Client.
+
+
+2. Navigate to the Datacenter and select the cluster you want to use for the installation. Right-click on the cluster and select **Deploy OVF Template**.
+
+
+3. Select the airgap OVA installer image you downloaded after receiving guidance from our support team.
+
+
+4. Select the folder where you want to install the Virtual Machine (VM) and assign a name to the VM.
+
+
+5. Next, select the compute resource.
+
+
+6. Review the details page. You may get a warning message stating the certificate is not trusted, you can ignore this message and select **Next**.
```

Comment:
```suggestion
6. Review the details page. You may get a warning message stating the certificate is not trusted, you can click **Ignore** the message and select **Next**.
```

Reply:
I modified this slightly because the user has to click the link to ignore it. Certificate errors could happen if they don't click Ignore, I think.